python/src/usecases/logistic_model.py

```diff
@@ -86,3 +88,25 @@ def __init__(self):
                 [2000, 281],
             ]
         )
+        self.data.setDescription(["Time", "U.S. Population"])
+        self.nbdates = self.data.getSize()
+
+        def logisticModel(X):
```

Comment:
This is a very much needed addition! I think if you add a docstring, it would help further document the class. Otherwise users might miss the existence of the method.

Reply:
I assume that you mention the new `data` attribute. I documented this new attribute in the docstring and I used the data set in the example of the calibration of the logistic model. What do you suggest to add? Would you print the `data` attribute within the example of the `LogisticModel`? 

That is possible (I can add that to other use cases as well).

Reply:
Actually, I meant the "logisticModel" method (sorry for the confusion). Is this method documented in the API section?

Reply:
Ok, I see. The function `logisticModel` is defined as a Python function, then converted to a `PythonFunction` and stored in the `model` attribute of the object. This attribute is documented in the docstring, which is presented in the API help page:

![image](https://user-images.githubusercontent.com/31351465/232200605-1ccd5b45-2202-4214-88aa-c8281073bebf.png)

This is similar to the current description of the other use cases. I think that you may want to see the input and output dimensions of the function is that it?

Reply:
I added a description of the data and model for Chaboche, Flood and logistic use cases.

Reply:
Here is the result for the flood model:

![image](https://user-images.githubusercontent.com/31351465/232208808-2bec84d7-eeb7-40c9-94bb-9791ffbee7ac.png)

For the Chaboche model:

![image](https://user-images.githubusercontent.com/31351465/232208835-74b6f173-ead2-40ce-a941-b69408d6d3bc.png)

For the Logistic model:

![image](https://user-images.githubusercontent.com/31351465/232208860-feb8b37d-4e7a-4a64-a797-acf5a42a95a1.png)